src/Sylius/Behat/Context/Setup/PaymentContext.php

```diff
@@ -158,8 +159,9 @@ private function createPaymentMethodFromNameAndCode($name, $code, $description =
         $paymentMethod->setDescription($description);
 
         if ($addForCurrentChannel && $this->sharedStorage->has('channel')) {
+            /** @var ChannelInterface $channel */
             $channel = $this->sharedStorage->get('channel');
```

Comment:
`$channel` variable is redundant. `$paymentMethod->addChannel($this->sharedStorage->get('channel'))`